pkgs/applications/science/math/scalp/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, stdenv
+, fetchgit
+, cmake
+, withGurobi ? false
+, gurobi
+, withCplex ? false
+, cplex
+, withLpsolve ? true
```

```suggestion
, gurobi
, cplex
```
Could we just activate everything by default?

---

Nope, the other solvers are commercial (therefore non-free) and not even cross-platform.